go.mod

```diff
@@ -5,65 +5,39 @@ go 1.16
 require (
 	github.com/99designs/gqlgen v0.13.0
 	github.com/agnivade/levenshtein v1.1.1 // indirect
-	github.com/aws/aws-sdk-go v1.17.7
-	github.com/cpuguy83/go-md2man v1.0.10 // indirect
 	github.com/dgrijalva/jwt-go v3.2.0+incompatible
 	github.com/disintegration/imaging v1.6.2
 	github.com/go-chi/chi v4.0.2+incompatible
-	github.com/gobuffalo/buffalo-plugins v1.13.1 // indirect
-	github.com/gobuffalo/events v1.4.1 // indirect
-	github.com/gobuffalo/flect v0.2.1 // indirect
+	github.com/gobuffalo/envy v1.9.0 // indirect
 	github.com/gobuffalo/genny v0.2.0 // indirect
-	github.com/gobuffalo/gitgen v0.0.0-20190315122116-cc086187d211 // indirect
 	github.com/gobuffalo/gogen v0.2.0 // indirect
-	github.com/gobuffalo/helpers v0.5.0 // indirect
-	github.com/gobuffalo/here v0.6.0 // indirect
-	github.com/gobuffalo/licenser v1.1.0 // indirect
 	github.com/gobuffalo/logger v1.0.3 // indirect
 	github.com/gobuffalo/mapi v1.2.1 // indirect
-	github.com/gobuffalo/meta v0.3.0 // indirect
 	github.com/gobuffalo/packd v1.0.0 // indirect
 	github.com/gobuffalo/packr/v2 v2.0.2
-	github.com/gobuffalo/plush v3.8.3+incompatible // indirect
-	github.com/gobuffalo/plushgen v0.1.2 // indirect
-	github.com/gobuffalo/release v1.7.0 // indirect
-	github.com/gobuffalo/shoulders v1.0.4 // indirect
-	github.com/gobuffalo/syncx v0.1.0 // indirect
 	github.com/gofrs/uuid v3.2.0+incompatible
 	github.com/golang-migrate/migrate/v4 v4.9.1
 	github.com/gorilla/sessions v1.1.3
-	github.com/gorilla/websocket v1.4.2
-	github.com/h2non/filetype v1.0.8
 	github.com/h2non/go-is-svg v0.0.0-20160927212452-35e8c4b0612c
 	github.com/hashicorp/golang-lru v0.5.4 // indirect
 	github.com/jmoiron/sqlx v1.2.0
 	github.com/karrick/godirwalk v1.16.1 // indirect
-	github.com/kisielk/errcheck v1.6.0 // indirect
-	github.com/konsorten/go-windows-terminal-sequences v1.0.3 // indirect
 	github.com/lib/pq v1.3.0
 	github.com/markbates/oncer v1.0.0 // indirect
-	github.com/matryer/moq v0.2.1 // indirect
-	github.com/mgechev/revive v1.0.6 // indirect
 	github.com/minio/minio-go/v7 v7.0.7
 	github.com/mitchellh/mapstructure v1.4.1 // indirect
 	github.com/pkg/errors v0.9.1
 	github.com/rogpeppe/go-internal v1.8.0 // indirect
 	github.com/rs/cors v1.6.0
-	github.com/russross/blackfriday/v2 v2.1.0 // indirect
 	github.com/sirupsen/logrus v1.8.1
-	github.com/spf13/cobra v1.1.3 // indirect
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.7.0
-	github.com/ugorji/go v1.1.4 // indirect
-	github.com/urfave/cli v1.20.0 // indirect
-	github.com/urfave/cli/v2 v2.3.0 // indirect
-	github.com/vektah/dataloaden v0.3.0 // indirect
```

Comment:
I've tried `go mod tidy` yesterday and it did exactly that.
This broke `make generate-dataloaders` for me (and maybe a few other things too, can't remember now).

Reply:
Guess I probably shouldn't have run tidy. In any case, readded the missing deps.

Reply:
oddly enough, `make generate-dataloaders` still fails.
```
cannot find package "." in:
        /path/to/stash-box/vendor/github.com/vektah/dataloaden
```

Reply:
Did you go mod vendor after pulling again?

Reply:
running `go install` seemed to fix it.